src/plone/restapi/services/locking/configure.zcml

```diff
@@ -1,17 +1,46 @@
 <configure
-    xmlns="http://namespaces.zope.org/zope"
-    xmlns:plone="http://namespaces.plone.org/plone"
-    xmlns:zcml="http://namespaces.zope.org/zcml"
-    >
+  xmlns="http://namespaces.zope.org/zope"
```

Comment:
Just wondering why zpretty is not complaining here? Either before or after. :)

Reply:
Fixed.